assets/xml/objects/object_fish.xml

```diff
@@ -1,162 +1,314 @@
+        <Texture Name="gFishingFishTopScalesTex" OutName="fishing_fish_dorsal_scales" Format="rgba16" Width="16" Height="16" Offset="0x16D0" />
+        <Texture Name="gFishingFishBottomScalesTex" OutName="fishing_fish_belly_scales" Format="rgba16" Width="16" Height="16" Offset="0x18D0" />
+        <Texture Name="gFishingFishHeadTex" OutName="fishing_fish_head" Format="rgba16" Width="32" Height="32" Offset="0x1AD0" />
+        <Texture Name="gFishingFishFinTex" OutName="fishing_fish_fin" Format="rgba16" Width="16" Height="16" Offset="0x22D0" />
+        <Texture Name="gFishingFishUnusedFinTex" OutName="fishing_fish_unused_fin" Format="rgba16" Width="16" Height="16" Offset="0x24D0" />
+        <Texture Name="gFishingFishSpinyFinTex" OutName="fishing_fish_spiny_fin" Format="rgba16" Width="16" Height="16" Offset="0x26D0" />
+
+        <!-- Fish Limbs and Skel -->
+        <Limb Name="gFishingFishHeadLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_HEAD" Offset="0x28D0" />
+        <Limb Name="gFishingFishTorsoRootLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_TORSO_ROOT" Offset="0x28DC" />
+        <Limb Name="gFishingFishTailRootLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_TAIL_ROOT" Offset="0x28E8" />
+        <Limb Name="gFishingFishEndRootLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_END_ROOT" Offset="0x28F4" />
+        <Limb Name="gFishingFishTailFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_TAIL_FIN" Offset="0x2900" />
+        <Limb Name="gFishingFishCaudalPeduncleEndLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_CAUDAL_PEDUNCLE_END" Offset="0x290C" />
+        <Limb Name="gFishingFishCaudalPeduncleLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_CAUDAL_PEDUNCLE" Offset="0x2918" />
+        <Limb Name="gFishingFishTopRearFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_TOP_REAR_FIN" Offset="0x2924" />
+        <Limb Name="gFishingFishBottomRearFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_BOTTOM_REAR_FIN" Offset="0x2930" />
+        <Limb Name="gFishingFishTorsoLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_TORSO" Offset="0x293C" />
+        <Limb Name="gFishingFishTopFrontFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_TOP_FRONT_FIN" Offset="0x2948" />
+        <Limb Name="gFishingFishBottomFrontFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_BOTTOM_FRONT_FIN" Offset="0x2954" />
+        <Limb Name="gFishingFishJawLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_JAW" Offset="0x2960" />
+        <Limb Name="gFishingFishRightPectoralFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_RIGHT_PECTORAL_FIN" Offset="0x296C" />
+        <Limb Name="gFishingFishLeftPectoralFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_LEFT_PECTORAL_FIN" Offset="0x2978" />
         <Skeleton Name="gFishingFishSkel" Type="Flex" LimbType="Standard" LimbNone="FISHING_FISH_LIMB_NONE" LimbMax="FISHING_FISH_LIMB_MAX" EnumName="FishingFishLimb" Offset="0x29C0" />
```

Comment:
I just wanted to transfer over another comment that was made here in a recent cleanup PR:
> You labeled the loach by species but this skeleton by "Fish", is this used by both the Bass and the Fry schools?

Did you notice this as well when looking at the fish?
https://github.com/zeldaret/mm/pull/1332#discussion_r1278508487

Reply:
idk i know `gFishingFish` is the bass but idk necessarily about the schools? is it maybe `gFishingGroupFish`? theres a comment saying its model dls original name is minnow_model idk
i have also thought of renaming `gFishingFish` to `gFishingBass` but i thoguht i might just leave it like that, i could rename it if its better that way

Reply:
The schools don't use skelAnime at all. They're just a static DL. From what I can tell, the bass and the loach use entirely different skeletons and animations. So you could name this `gFishingBass` instead and it would be fine. I think that's probably preferable so that the two different species are very distinct.